README.md

```diff
@@ -1 +1,215 @@
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="my_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello_Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_Microverse_pro<a name="about-project"></a>
+
+>This is Hello_Microverse_pro I created it using HTML, CSS and JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
+3- Liners.
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- added index.html file.
+- added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> coming soon
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. web browser.
+2. code editor.
+3. Git-smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+ Run this comand:
+
+```sh
+  cd my-folder
+  git clone https://github.com/mhnazary/Hello-Microverse.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+Run this comand:
+
+```sh
+  cd my-project
+  npm install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension
+
+### Run tests
+
+> coming soon
+
+### Deployment
+
+> coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] Kindly consider removing this line as it is referred to the author of this repo and not to the visitor. removing it will make your repo look more professional.